requirements.txt

```diff
@@ -146,7 +146,7 @@ grpcio==1.63.0
     #   grpcio-status
 grpcio-status==1.48.2
     # via google-api-core
-hail==0.2.130
+hail==0.2.132
```

Comment:
There was a [hail bug](https://hail.zulipchat.com/#narrow/stream/123010-Hail-Query-0.2E2-support/topic/.E2.9C.94.20Containerised.20Local.20Spark.20Instance) blocking the docker container from working

Reply:
We are using the [hail dockerfiles](https://github.com/broadinstitute/seqr-loading-pipelines/pull/841/files/dd87fef8cb96ac9d1a6874566ac795c193f58f00#diff-2d6f786f785631b3bb36f4899641a7b6e5ff13af20730359a3e41725fff5732aR2).  This is the requirements file for "our" requirements that the tests use (and in theory our dataproc deployment 🤦 ).  The only reason for the change here is parity.